### DISCUSSION TITLE
Battery Impact of Decks

### DISCUSSION THREAD

**Discussion by mbaytas** [bitcraze/discussions#604] [Q&A] [answered]
I am wondering if anyone has data on the battery life impact of various decks.

(Specifically I'm interested in the Qualisys active marker deck, but curious about the other decks as well.)

My estimate is that the battery life impact of any deck is **negligible** compared to the moving parts. I'd appreciate comments that support or refute this estiamte.

Context: I work with Qualisys. We have many customers in engineering research who use Qualisys systems and Crazyflie drones together. This is a question we received from one of those potential customers, yet we don't have good data. I hope that the Bitcraze community can provide more precise insights.

**Comment by ataffanel** [comment, marked as answer]
The motors of the Crazyflie consume about 8W of power in flight. The most power hungry deck, the LED ring, consumes about 1W so it is not completely negligible but it is 1/8th of the power of the motors. The Active marker deck though does not consume much power at all since the LEDs on it are flashing rather than saying ON all the time.

We do not have precise data on the decks battery impact unfortunately but our feeling is that weight will, in most case, be the main factor and not the power consumption.
